expyfun/codeblocks/_pupillometry.py

```diff
@@ -194,13 +198,13 @@ def find_pupil_tone_impulse_response(ec, el, bgcolor, fcolor, prompt=True,
     #
     fs = ec.stim_fs
     n_samp = int(fs * stim_dur)
-    freqs = np.ones(n_samp) * f0
     t = np.arange(n_samp).astype(float) / fs
-    tone_stim = np.sin(2 * np.pi * freqs * t)
-    freqs = 100 * np.sin(2 * np.pi * (1 / stim_dur) * t) + f0
-    sweep_stim = np.sin(2 * np.pi * np.cumsum(freqs) / fs)
+    steady = np.sin(2 * np.pi * f0 * t)
+    wobble = np.sin(np.cumsum(f0 + 100 * np.sin(2 * np.pi * (1 / stim_dur) * t)
+                              ) / fs * 2 * np.pi)
+    tone_stim, targ_stim = (steady, wobble) if targ_is_fm else (wobble, steady)
```

Comment:
yeah, that makes sense. was not really satisfied with my naming choice to begin with.